internal/spdxlicense/generate/generate_license_list.go

```diff
@@ -37,16 +37,46 @@ var licenseIDs = map[string]string{
 
 var versionMatch = regexp.MustCompile(`-([0-9]+)\.?([0-9]+)?\.?([0-9]+)?\.?`)
 
+type License struct {
+	ID          string   `json:"licenseId"`
+	Name        string   `json:"name"`
+	Text        string   `json:"licenseText"`
+	Deprecated  bool     `json:"isDeprecatedLicenseId"`
+	OSIApproved bool     `json:"isOsiApproved"`
+	SeeAlso     []string `json:"seeAlso"`
+}
+
+func (l License) isEquivalent(other License) bool {
+	// With this equality check we guarantee the call order doesn't matter:
+	// a.isEquivalent(b) AND b.isEquivalent(a) yield the same value
+	if l.Deprecated == other.Deprecated {
```

Comment:
this is really a concern of the caller and not within a function named "isEquivelent" (where deprecation has no barring on equivalency)